packages/breadboard-web/src/boards/open-alex-entity-search-list.ts

```diff
@@ -4,177 +4,108 @@
-import { core } from "@google-labs/core-kit";
+import {
+  board,
+  enumeration,
+  input,
+  object,
+  output,
+} from "@breadboard-ai/build";
+import { fetch, code } from "@google-labs/core-kit";
 
-import { templates } from "@google-labs/template-kit";
+import { urlTemplate } from "@google-labs/template-kit";
 
-const spread = code<{ object: object }>((inputs) => {
-  const object = inputs.object;
-  if (typeof object !== "object") {
-    throw new Error(`object is of type ${typeof object} not object`);
-  }
-  return { ...object };
-});
-
-const openAlexEntities: Schema = {
+const entity = input({
   description: "Entity type to search for",
-  type: "string",
-  default: "works",
-  enum: [
-    "works",
+  type: enumeration("works",
     "authors",
     "sources",
     "institutions",
     "topics",
     "publishers",
     "funders",
-    "concepts",
-  ],
+    "concepts"),
+  default: "works",
   title: "Entity",
-};
-
-const pageInputSchema: Schema = {
-  type: "integer",
-  default: "1",
-  title: "Page number",
-  description: "Page number to return",
-};
+});
 
-const perPageInputSchema: Schema = {
-  type: "integer",
+const page = input({
   default: "5",
   title: "Results per page",
   description: "Number of results to return per page",
-};
+});
 
-const selectInputSchema: Schema = {
-  type: "string",
-  title: "Select",
-  default: "id,display_name,title,relevance_score",
-  description: "Comma-separated list of fields to return",
-};
+const per_page = input({
+  default: "5",
+  title: "Results per page",
+  description: "Number of results to return per page",
+});
 
-const searchInputSchema = {
+const search = input({
   type: "string",
   title: "Search term",
   default: "Artificial Intelligence",
   description: "Search term to search for, double quotes for exact match",
-};
-
-const graph = board(() => {
-  const input = base.input({
-    $id: "query",
-    schema: {
-      title: "OpenAlex Search",
-      properties: {
-        search: searchInputSchema,
-        page: pageInputSchema,
-        per_page: perPageInputSchema,
-        entity: openAlexEntities,
-        select: selectInputSchema,
-      },
-      type: "object",
-      required: ["search"],
-      additionalProperties: false,
-    },
-  });
+});
 
-  const urlTemplate = templates.urlTemplate({
-    $id: "urlTemplate",
-    template:
-      "https://api.openalex.org/{entity}?search={search}&page={page}&per_page={per_page}&select={select}",
-    entity: input.entity,
-    page: input.page,
-    per_page: input.per_page,
-    search: input.search,
-    select: input.select,
-  });
+const select = input({
+  type: "string",
+  title: "Select",
+  default: "id,display_name,title,relevance_score",
+  description: "Comma-separated list of fields to return",
+});
 
-  const fetchUrl = core.fetch({
-    $id: "fetch",
-    method: "GET",
-    url: urlTemplate.url,
-  });
+const urlTemplater = urlTemplate({
```

Comment:
Suggest calling this variable `url`.

(In general, I personally find it a bit more intuitive to name output values like this after the thing that they *are*, rather than *the thing that made them*, if that makes sense?)